fhir-config/src/main/java/com/ibm/fhir/config/FHIRRequestContext.java

```diff
@@ -137,6 +140,24 @@ public void setDataStoreId(String dataStoreId) throws FHIRException {
         }
     }
 
+    /**
+     * set an Operation Context property
+     * @param name
+     * @param value
+     */
+    public void setExtendedOperationProperties(String name, Object value) {
```

Comment:
remind me what these are for again?

Reply:
To pack the properties that are used in an Operation from a Bundle.  the RequestContext is the only way to pass it without massively changing the signatures of FHIRRestHelpers